src/applications/mhv-medical-records/components/DownloadRecords/DownloadDateRange.jsx

```diff
@@ -190,12 +204,22 @@ const DownloadDateRange = () => {
               data-testid="va-date-end-date"
               value={customToDate}
               onDateChange={e => {
+                const val = e.target.value;
                 const [year, month, day] = e.target.value.split('-');
                 if (parseInt(year, 10) >= 1900 && month && day) {
                   setCustomToError(null);
-                  setCustomToDate(e.target.value);
+                  setCustomToDate(val);
                 }
               }}
+              invalidDay={() => {
+                return customToError === ERROR_END_AFTER_START_DATE;
+              }}
+              invalidMonth={() => {
+                return customToError === ERROR_END_AFTER_START_DATE;
+              }}
+              invalidYear={() => {
+                return customToError === ERROR_END_AFTER_START_DATE;
+              }}
```

Comment:
Maybe define a function above and just use it three times, rather than repeat the code.